buildenv

```diff
@@ -31,11 +31,82 @@ zopen_check_results()
 zopen_append_to_env()
 {
   # echo envars outside of PATH, MANPATH, LIBPATH
+
+  # i'm not seeing these prints in zopen build.
+  # perhaps its written the setup.sh in the 
+  # usr/local/zopen dir
+
+  echo "***** in append_to_env() start"
+  echo "CWD: "
+  pwd
+
+
+
+  # update the path for depot_tools
+  export PATH=${PATH}:${HOME}/zopen/dev/depot_toolsport/depot_tools
```

Comment:
zopen build automatically adds the install path + "/bin" to the PATH envrionment variable